lib/datasource/maven/index.spec.ts

```diff
@@ -1,314 +1,315 @@
-import nock from 'nock';
-import { Release, getPkgReleases } from '..';
-import { getName, loadFixture } from '../../../test/util';
+import _fs from 'fs-extra';
```

Comment:
Isn't the code using our `utils/fs` wrapper?

Reply:
Won't work since `maven/util.ts` uses `fs-extra` directly

Reply:
ok, so needs refactored /fixed in separate pr.